pagopa/pn-frontend
# Maintenance flag - pn-4427

## Short description
Added the WORK_IN_PROGRESS configuration to show the AppNotAccessible

## List of changes proposed in this pull request
- added configuration in configuration.service.ts (PF/PA/PG)
- use that configuration in SessionGuard.tsx (PF/PA/PG)
- updated tests

## How to test
Add WORK_IN_PROGRESS: true in local config file -> check that the app is not accessible if there is no user logged in
